src/components/Events/styles.module.css

```diff
@@ -1,161 +1,223 @@
+  -moz-user-select: none;
+  /* Old versions of Firefox */
+  -ms-user-select: none;
+  /* Internet Explorer/Edge */
+  user-select: none;
+  /* Non-prefixed version, currently supported by Chrome, Edge, Opera and Firefox */
+}
+
+.event:hover h3 {
+  color: white;
+  transition: 0.5s;
+  overflow: hidden;
+}
+
+.event a {
+  font-family: Raleway, serif;
+  font-size: 1rem;
+  background-color: var(--events-color);
+  color: #fff;
+  padding: 10px;
+  font-weight: 400;
+  transition: 0.5s;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  border-radius: 0.5rem;
+  align-content: center;
+  line-height: 1.5em;
+  margin: 0.16em;
+  border: 1px solid var(--events-color);
+  box-shadow: var(--events-color);
+  box-sizing: content-box;
+  /* disable text selection */
+  -webkit-touch-callout: none;
+  /* iOS Safari */
+  -webkit-user-select: none;
+  /* Safari */
+  -khtml-user-select: none;
+  /* Konqueror HTML */
+  -moz-user-select: none;
+  /* Old versions of Firefox */
+  -ms-user-select: none;
+  /* Internet Explorer/Edge */
+  user-select: none;
+  /* Non-prefixed version, currently supported by Chrome, Edge, Opera and Firefox */
+  text-decoration: none;
+}
+
+.event:hover a {
+  color: var(--events-color);
+  transition: 0.5s;
+  background-color: #fff;
 }
 
 .event p {
-    font-family: Raleway, serif;
-    font-size: 1rem;
-    color: var(--events-color);
-    font-weight: 400;
-    transition: .5s;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow: hidden;
-
-    /* disable text selection */
-    -webkit-touch-callout: none; /* iOS Safari */
-    -webkit-user-select: none; /* Safari */
-    -khtml-user-select: none; /* Konqueror HTML */
-    -moz-user-select: none; /* Old versions of Firefox */
-    -ms-user-select: none; /* Internet Explorer/Edge */
-    user-select: none; /* Non-prefixed version, currently supported by Chrome, Edge, Opera and Firefox */
-}
-
-.event:hover p{
-    color: white;
-    transition: .5s;
+  font-family: Raleway, serif;
+  font-size: 1rem;
+  color: var(--events-color);
+  font-weight: 400;
+  transition: 0.5s;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+
+  /* disable text selection */
+  -webkit-touch-callout: none;
+  /* iOS Safari */
+  -webkit-user-select: none;
+  /* Safari */
+  -khtml-user-select: none;
+  /* Konqueror HTML */
+  -moz-user-select: none;
+  /* Old versions of Firefox */
+  -ms-user-select: none;
+  /* Internet Explorer/Edge */
+  user-select: none;
+  /* Non-prefixed version, currently supported by Chrome, Edge, Opera and Firefox */
+}
+
+.event:hover p {
+  color: white;
+  transition: 0.5s;
+}
+
+.eventLinks {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 1rem 0 0;
+}
+
+.addToCalendarButton {
+  padding: 0.7rem;
+  background-color: var(--secondary-color);
+  color: var(--events-color);
+}
+
+.addToCalendarButton:hover {
+  color: var(--events-color);
```

Comment:
wonder if the add to calendar button text color has bad contrast, we can change this to be --secondary-color instead of events-color? wdyt 

with --events-color
![image](https://github.com/San-Antonio-ACM-Chapter/san-antonio-acm-chapter.github.io/assets/61089246/09ddd86e-0213-466e-8479-640b21a84b0d)




with --secondary-color
![image](https://github.com/San-Antonio-ACM-Chapter/san-antonio-acm-chapter.github.io/assets/61089246/c6322650-367d-4e3c-a895-694f1276b383)

Reply:
I personally am not a fan of the black on blue look. the use of `--secondary-color` is fine. But maybe we should choose a different color for the background of the button. maybe a step or two darker than the primary blue that is used. thoughts?